systemvm/js/ajaxviewer.js

```diff
@@ -1347,17 +1358,17 @@ AjaxViewer.prototype = {
 	onMouseUp: function(x, y, whichButton, modifiers) {
 		this.sendMouseEvent(AjaxViewer.MOUSE_UP, x, y, whichButton, modifiers);
 		
-		var curTick = new Date().getTime();
+		/*var curTick = new Date().getTime();
```

Comment:
Please do not leave commented code behind.
If a code is not used anymore delete it.